cyral/resource_cyral_policy_rule_test.go

```diff
@@ -29,30 +30,70 @@ var initialPolicyRuleConfig PolicyRuleConfig = PolicyRuleConfig{
 	DeletedRateLimit: 1,
 	UpdatedRateLimit: 2,
 	ReadRateLimit:    3,
+	Identities: &Identity{
+		DBRoles: []string{
+			"db-role-1",
+		},
+	},
 }
 
-var updated1PolicyRuleConfig PolicyRuleConfig = PolicyRuleConfig{
+var updatedGroupsPolicyRuleConfig PolicyRuleConfig = PolicyRuleConfig{
 	DeletedSeverity:  "low",
 	UpdatedSeverity:  "medium",
 	ReadSeverity:     "high",
 	DeletedRateLimit: 2,
 	UpdatedRateLimit: 3,
 	ReadRateLimit:    4,
+	Identities: &Identity{
+		Groups: []string{
+			"groups-1",
+		},
+	},
 }
 
-var updated2PolicyRuleConfig PolicyRuleConfig = PolicyRuleConfig{
+var updatedServicesPolicyRuleConfig PolicyRuleConfig = PolicyRuleConfig{
 	DeletedSeverity:  "high",
 	UpdatedSeverity:  "low",
 	ReadSeverity:     "medium",
 	DeletedRateLimit: 5,
 	UpdatedRateLimit: 6,
 	ReadRateLimit:    7,
+	Identities: &Identity{
+		Services: []string{
+			"services-1",
+		},
+	},
+}
+
+var updatedUsersPolicyRuleConfig PolicyRuleConfig = PolicyRuleConfig{
+	DeletedSeverity:  "high",
+	UpdatedSeverity:  "low",
+	ReadSeverity:     "medium",
+	DeletedRateLimit: 5,
+	UpdatedRateLimit: 6,
+	ReadRateLimit:    7,
+	Identities: &Identity{
+		Users: []string{
+			"users-1",
+		},
+	},
+}
+
+var updatedNoIdentityPolicyRuleConfig PolicyRuleConfig = PolicyRuleConfig{
+	DeletedSeverity:  "low",
+	UpdatedSeverity:  "medium",
+	ReadSeverity:     "high",
+	DeletedRateLimit: 1,
+	UpdatedRateLimit: 2,
+	ReadRateLimit:    3,
 }
 
 func TestAccPolicyRuleResource(t *testing.T) {
 	testConfig, testFunc := setupPolicyRuleTest(initialPolicyRuleConfig)
-	testUpdate1Config, testUpdate1Func := setupPolicyRuleTest(updated1PolicyRuleConfig)
-	testUpdate2Config, testUpdate2Func := setupPolicyRuleTest(updated2PolicyRuleConfig)
+	testGroupsConfig, testGroupsFunc := setupPolicyRuleTest(updatedGroupsPolicyRuleConfig)
+	testServicesConfig, testServicesFunc := setupPolicyRuleTest(updatedServicesPolicyRuleConfig)
+	testUsersConfig, testUsersFunc := setupPolicyRuleTest(updatedServicesPolicyRuleConfig)
```

Comment:
I believe this is supposed to be `updatedUsersPolicyRuleConfig`, right?
```suggestion
	testUsersConfig, testUsersFunc := setupPolicyRuleTest(updatedUsersPolicyRuleConfig)
```